index.js

```diff
@@ -0,0 +1,16 @@
+const redBlock = document.querySelector(".block--red");
+const blueBlock = document.querySelector(".block--blue");
+const greenBlock = document.querySelector(".block--green");
+const pinkBlock = document.querySelector(".block--pink");
+const grayBlock = document.querySelector(".block--gray");
+
+let grayBlockTraveler = grayBlock.style.marginLeft;
+
+grayBlock.addEventListener("mousedown", (event) =>{
+        grayBlock.style.transform = "translate(900px, 0%)"
```

Comment:
If the element travels too far, as in this case with 900px, `mouseup` won't work properly. Also, if the second parameter in `translate()` is 0, you can just leave it out.